ui/tests/pages/allocations/task/fs.js

```diff
@@ -1,12 +1,50 @@
-import { create, isPresent, text, visitable } from 'ember-cli-page-object';
+import {
+  attribute,
+  collection,
+  clickable,
+  create,
+  hasClass,
+  isPresent,
+  text,
+  visitable,
+} from 'ember-cli-page-object';
 
 export default create({
   visit: visitable('/allocations/:id/:name/fs'),
+  visitPath: visitable('/allocations/:id/:name/fs/:path'),
 
-  hasFiles: isPresent('[data-test-file-explorer]'),
+  fileViewer: {
+    scope: '[data-test-file-viewer]',
+  },
+
+  breadcrumbsText: text('[data-test-fs-breadcrumbs]'),
+
+  breadcrumbs: collection('[data-test-fs-breadcrumbs] li', {
+    visit: clickable('a'),
+    path: attribute('href', 'a'),
```

Comment:
Oh, this helps clear up what the assertions I was confused about are for. I think it's still worth adding an assertion description though.